labelbox/client.py

```diff
@@ -434,16 +437,15 @@ def _create(self, db_object_type, data):
         return db_object_type(self, res)
 
     def create_dataset(self, **kwargs):
-        """ Creates a Dataset object on the server. Attribute values are
-            passed as keyword arguments:
+        """ Creates a Dataset object on the server.
+            
+        Attribute values are passed as keyword arguments.
 
-            >>> project = client.get_project("<project_uid>")
-            >>> dataset = client.create_dataset(name="<dataset_name>", projects=project)
+        >>> project = client.get_project("<project_uid>")
+        >>> dataset = client.create_dataset(name="<dataset_name>", projects=project)
 
-        Kwargs:
-            Keyword arguments with new Dataset attribute values.
-            Keys are attribute names (in Python, snake-case convention) and
-            values are desired attribute values.
+        Args:
+            kwargs: Keyword arguments with Dataset attribute values.
```

Comment:
Are you sure this generates the correct thing? `kwargs` is not like other, normal args... It should be documented differently, but I'm not exactly sure which convetion you/Sphynx prefer.

Some quick googling gives: https://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter

Reply:
Fixed the formatting for kwargs. Decided to go with [this format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) so we can leave kwargs unexpanded. Sphinx recognized Args as an alias of Parameters, [see docs](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#sections). Decided to keep Parameters throughout for consistency.

Reply:
Agreed regarding `kwargs`.

I don't understand what you mean by "decided to keep Parameters throughout for consistency"... We're not using "Parameters" in the documentation, but "Args"?

Reply:
"Args" and "Arguments" are aliases for "Parameters". So, rather than replacing all instances of "Args" in our current documentation, I just kept "Args" as is and indicate all instances of kwargs with a double asterisk. When Sphinx generates the docs, all instances of "Args" become "Parameters". So, it will look like this:
<img width="718" alt="Screen Shot 2021-01-14 at 8 47 53 AM" src="https://user-images.githubusercontent.com/25472296/104622397-0d8d5e00-5646-11eb-84eb-45edbbbb02bf.png">

Reply:
OK, got it.